apps/heft/src/cli/actions/TestAction.ts

```diff
@@ -110,6 +111,17 @@ export class TestAction extends BuildAction {
         ' default reporter would have presented it. Include this output in your bug report.' +
         ' Do not use "--debug-heft-reporter" in production.'
     });
+
+    this._maxWorkers = this.defineStringParameter({
+      parameterLongName: '--max-workers',
+      argumentName: 'STRING',
```

Comment:
MAX_WORKERS

Reply:
It should probably be `COUNT`, and this parameter should probably be an integer parameter.

```suggestion
      argumentName: 'COUNT',
```

Reply:
Or perhaps

```suggestion
      argumentName: 'COUNT_OR_PERCENTAGE',
```